docs/en/reference/events.rst

```diff
@@ -121,6 +121,16 @@ Now you can test the ``$eventSubscriber`` instance to see if the
         echo 'pre foo invoked!';
     }
 
+Registering Events
+~~~~~~~~~~~~~~~~~~
+
+There are two ways to register an event:
+
+* *All events* can be registered by calling ``$eventManager->addEventListener()``
+or ``eventManager->addEventSubscriber()`` (see [below](#listening-and-subscribing-to-lifecycle-events))
```

Comment:
This syntax looks like markdown… does it really work in RST?

Reply:
Thanks! Please see https://github.com/doctrine/orm/pull/9102